### PR TITLE
Renovate configuration fixes

### PR DESCRIPTION
**Pull request by kaworu**
Bundle of Renovate config fixes.

I pushed this patch to https://github.com/kaworu/cilium `main` branch and ran:

```console
% npx --package=renovate -c renovate\ --dry-run\ --token\ $GITHUB_TOKEN\ kaworu/cilium
```
<details>
 <summary>command output</summary>

```
 WARN: cli config dryRun property has been changed to full
 WARN: Please upgrade the version of Node.js used to run Renovate to satisfy "^18.12.0 || >=20.0.0". Support for your current version will be removed in Renovate's next major release.
       "versions": {
         "node": "19.9.0",
         "acorn": "8.8.2",
         "ada": "1.0.4",
         "ares": "1.19.0",
         "brotli": "1.0.9",
         "cldr": "42.0",
         "icu": "72.1",
         "llhttp": "8.1.0",
         "modules": "111",
         "napi": "8",
         "nghttp2": "1.52.0",
         "openssl": "3.0.8",
         "simdutf": "3.2.3",
         "tz": "2022e",
         "undici": "5.21.0",
         "unicode": "15.0",
         "uv": "1.44.2",
         "uvwasi": "0.0.16",
         "v8": "10.8.168.25-node.16",
         "zlib": "1.2.13"
       }
 INFO: Repository started (repository=kaworu/cilium)
       "renovateVersion": "35.74.1"
 INFO: git fetch completed (repository=kaworu/cilium)
       "durationMs": 3622
 INFO: Cannot ensure issue because issues are disabled in this repository (repository=kaworu/cilium)
 INFO: Dependency extraction complete (repository=kaworu/cilium, baseBranch=main)
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 8, "depCount": 29},
           "github-actions": {"fileCount": 62, "depCount": 570},
           "gomod": {"fileCount": 1, "depCount": 249},
           "kubernetes": {"fileCount": 1, "depCount": 2},
           "regex": {"fileCount": 50, "depCount": 59}
         },
         "total": {"fileCount": 122, "depCount": 909}
       }
 INFO: Dependency extraction complete (repository=kaworu/cilium, baseBranch=v1.13)
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 8, "depCount": 28},
           "github-actions": {"fileCount": 18, "depCount": 125},
           "gomod": {"fileCount": 1, "depCount": 241},
           "kubernetes": {"fileCount": 1, "depCount": 2},
           "regex": {"fileCount": 1, "depCount": 3}
         },
         "total": {"fileCount": 29, "depCount": 399}
       }
 INFO: Dependency extraction complete (repository=kaworu/cilium, baseBranch=v1.12)
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 8, "depCount": 28},
           "github-actions": {"fileCount": 16, "depCount": 104},
           "gomod": {"fileCount": 2, "depCount": 226},
           "kubernetes": {"fileCount": 1, "depCount": 2},
           "regex": {"fileCount": 1, "depCount": 3}
         },
         "total": {"fileCount": 28, "depCount": 363}
       }
 INFO: Dependency extraction complete (repository=kaworu/cilium, baseBranch=v1.11)
       "stats": {
         "managers": {
           "dockerfile": {"fileCount": 8, "depCount": 28},
           "github-actions": {"fileCount": 15, "depCount": 101},
           "gomod": {"fileCount": 2, "depCount": 221},
           "kubernetes": {"fileCount": 1, "depCount": 2},
           "regex": {"fileCount": 1, "depCount": 3}
         },
         "total": {"fileCount": 27, "depCount": 355}
       }
 INFO: GitHub GraphQL datasource: skipping empty item (repository=kaworu/cilium, baseBranch=main)
       "packageName": "coreos/go-systemd",
       "baseUrl": "https://api.github.com/"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "github.com/aws/aws-sdk-go-v2",
       "currentValue": "v1.17.8",
       "previousNewValue": "v1.18.0",
       "thisNewValue": "v1.17.8"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "github.com/go-openapi/runtime",
       "currentValue": "v0.25.0",
       "previousNewValue": "v0.26.0",
       "thisNewValue": "v0.25.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "github.com/osrg/gobgp/v3",
       "currentValue": "v3.13.0",
       "previousNewValue": "v3.14.0",
       "thisNewValue": "v3.13.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "github.com/prometheus/client_model",
       "currentValue": "v0.3.0",
       "previousNewValue": "v0.4.0",
       "thisNewValue": "v0.3.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "go.opentelemetry.io/otel",
       "currentValue": "v1.14.0",
       "previousNewValue": "v1.15.1",
       "thisNewValue": "v1.14.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "go.opentelemetry.io/otel/trace",
       "currentValue": "v1.14.0",
       "previousNewValue": "v1.15.1",
       "thisNewValue": "v1.14.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "go.uber.org/dig",
       "currentValue": "v1.16.1",
       "previousNewValue": "v1.17.0",
       "thisNewValue": "v1.16.1"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "golang.org/x/crypto",
       "currentValue": "v0.8.0",
       "previousNewValue": "v0.9.0",
       "thisNewValue": "v0.8.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "golang.org/x/net",
       "currentValue": "v0.9.0",
       "previousNewValue": "v0.10.0",
       "thisNewValue": "v0.9.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "golang.org/x/sync",
       "currentValue": "v0.1.0",
       "previousNewValue": "v0.2.0",
       "thisNewValue": "v0.1.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "golang.org/x/sys",
       "currentValue": "v0.7.0",
       "previousNewValue": "v0.8.0",
       "thisNewValue": "v0.7.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "golang.org/x/term",
       "currentValue": "v0.7.0",
       "previousNewValue": "v0.8.0",
       "thisNewValue": "v0.7.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "golang.org/x/tools",
       "currentValue": "v0.8.0",
       "previousNewValue": "v0.9.0",
       "thisNewValue": "v0.8.0"
 INFO: Ignoring upgrade collision (repository=kaworu/cilium, baseBranch=main, packageFile=go.mod, branch=renovate/main-all-go-deps-main)
       "manager": "gomod",
       "depName": "google.golang.org/grpc",
       "currentValue": "v1.54.0",
       "previousNewValue": "v1.55.0",
       "thisNewValue": "v1.54.0"
 INFO: DRY-RUN: Would commit files to branch renovate/main-patch-hubble-cli (repository=kaworu/cilium, baseBranch=main, branch=renovate/main-patch-hubble-cli)
 INFO: DRY-RUN: Would commit files to branch renovate/v1.13-patch-hubble-cli (repository=kaworu/cilium, baseBranch=v1.13, branch=renovate/v1.13-patch-hubble-cli)
 INFO: DRY-RUN: Would commit files to branch renovate/v1.12-patch-hubble-cli (repository=kaworu/cilium, baseBranch=v1.12, branch=renovate/v1.12-patch-hubble-cli)
 INFO: DRY-RUN: Would commit files to branch renovate/v1.11-patch-hubble-cli (repository=kaworu/cilium, baseBranch=v1.11, branch=renovate/v1.11-patch-hubble-cli)
 INFO: DRY-RUN: Would ensure Dependency Dashboard (repository=kaworu/cilium)
       "title": "Dependency Dashboard"
 INFO: DRY-RUN: Would save repository cache. (repository=kaworu/cilium)
 INFO: Cannot ensure issue because issues are disabled in this repository (repository=kaworu/cilium)
 INFO: Repository finished (repository=kaworu/cilium)
       "cloned": true,
       "durationMs": 954738
```
</details>

Unfortunately I don't understand why Renovate would not open PRs for
1. [contrib/backporting/Dockerfile](https://github.com/cilium/cilium/blob/cef9595361e970dbee47eed659bac56ea68d73cd/contrib/backporting/Dockerfile#L1) from golang 1.20.2 to 1.20.4
2. [contrib/coccinelle/Dockerfile](https://github.com/cilium/cilium/blob/cef9595361e970dbee47eed659bac56ea68d73cd/contrib/coccinelle/Dockerfile#L1) from alpine 3.16.0 to alpine 3.17.3
3. [contrib/coccinelle/Dockerfile](https://github.com/cilium/cilium/blob/8f2445df6ac5f7fb150e8a610ee31b1bac8da57a/contrib/coccinelle/Dockerfile#L1), [images/operator/Dockerfile](https://github.com/cilium/cilium/blob/8f2445df6ac5f7fb150e8a610ee31b1bac8da57a/images/operator/Dockerfile#L6), [images/cache/Dockerfile](https://github.com/cilium/cilium/blob/8f2445df6ac5f7fb150e8a610ee31b1bac8da57a/images/cache/Dockerfile#L4,L18), [images/hubble-relay/Dockerfile](https://github.com/cilium/cilium/blob/8f2445df6ac5f7fb150e8a610ee31b1bac8da57a/images/hubble-relay/Dockerfile#L6), and [images/clustermesh-apiserver/Dockerfile](https://github.com/cilium/cilium/blob/8f2445df6ac5f7fb150e8a610ee31b1bac8da57a/images/clustermesh-apiserver/Dockerfile#L6) from alpine 3.17.0 to alpine 3.17.3

### CODE REVIEW

**Comment by kaworu** [comment]
Ok seems working to me at https://github.com/kaworu/cilium/issues/292, marking as ready for review.

**Review by JAORMX — APPROVED** [review]

File: .github/renovate.json5 [hunk 1]
```diff
@@ -341,8 +341,8 @@
     },
     {
       "fileMatch": [
-	"^test/kubernetes-test\\.sh$",
```
Comment (bimmlerd) [inline .github/renovate.json5:344]: nit: is this intentional?

**Review by brlbil — APPROVED** [review]

**Review by brlbil — APPROVED** [review]

**Review by bimmlerd — APPROVED** [review]
lgtm, minor nit

File: .github/renovate.json5 (hunk 1 above)
Comment (kaworu) [reply .github/renovate.json5:344]: Yes, those two lines had tabs as indent which was inconsistent with the rest of the file, so I replaced them with spaces on the way.

**Comment by kaworu** [comment]
All reviews are in and this patch doesn't need a full CI run, so marking as `ready-to-merge`

**Pull request merged**